src/test/java/com/revature/testing/FocusTest.java

```diff
@@ -17,6 +27,9 @@
 @RunWith(SpringJUnit4ClassRunner.class)
 @SpringBootTest
 public class FocusTest {
```

Comment:
Standardize indentation in this file.  I see 3 different ways of indenting at the same depth.